ext/standard/dir.stub.php

```diff
@@ -2,6 +2,80 @@
 
 /** @generate-class-entries */
 
+#ifdef HAVE_GLOB
+#if defined(GLOB_BRACE) && (GLOB_BRACE != 0)
```

Comment:
In fact, I mean this:
```suggestion
#if (defined(GLOB_BRACE) && GLOB_BRACE != 0)
```

This way the following arginfo code would be generated:
```
#if defined(HAVE_GLOB) && (defined(GLOB_BRACE) && GLOB_BRACE != 0)
```

Reply:
Corrected. Sorry for making a mess of this PR.